### PR TITLE
[Actions] Add a rebase action.

### PR DESCRIPTION
**Pull request by mandel-macaque**
As part of the effort to move away from jenkins, the @monojenkins bot will be retired. This adds an action to be used that does the same job.

The new action can be executed via '@vs-mobiletools-engineering-service2 rebase'

### CODE REVIEW

**Review by dalexsoto — APPROVED** [review]
ugh `@vs-mobiletools-engineering-service2` is mouthful can we rename it?

File: .github/workflows/rebase-trigger.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: Rebase Trigger
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  launchBackportBuild:
+    runs-on: ubuntu-latest
+    if: github.event.issue.pull_request != '' && contains(github.event.comment.body, '@vs-mobiletools-engineering-service2 rebase')
```
Comment (rolfbjarne) [inline .github/workflows/rebase-trigger.yml:10]: Would this work?

```suggestion
    if: github.event.issue.pull_request != '' && contains(github.event.comment.body, '@monojenkins rebase')
```

Comment (mandel-macaque) [reply .github/workflows/rebase-trigger.yml:10]: Lets try /sudo

**Review by tj-devel709 — APPROVED** [review]

**Comment by xamarin-release-manager** [comment]
❌ [Build was (probably) aborted](https://jenkins.internalx.com/job/macios/job/PR-10259/1/display/redirect)

🔥 [Jenkins job](https://jenkins.internalx.com/job/macios/job/PR-10259/1/display/redirect) (on internal Jenkins) failed in stage(s) 'Checkout' 🔥 : hudson.AbortException: script returned exit code 128

🔥 [Provisioning failed](https://jenkins.internalx.com/job/macios/job/main/604//console) 🔥
🔥 [Failed to publish results](https://jenkins.internalx.com/job/macios/job/main/604//console) 🔥

**Pull request merged**